src/main/kotlin/no/nav/familie/ef/sak/service/steg/BeslutteVedtakSteg.kt

```diff
@@ -37,12 +39,12 @@ class BeslutteVedtakSteg(private val taskRepository: TaskRepository,
         ferdigstillOppgave(behandling)
 
         return if (data.godkjent) {
-            // TODO oppdater brev
-                if (behandling.type != BehandlingType.BLANKETT) {
-                    opprettTaskForIverksettMotOppdrag(behandling)
-                } else {
-                    opprettTaskForJournalførBlankett(behandling)
-                }
+            vedtaksbrevService.lagreEndeligBrev(behandling.id)
```

Comment:
Er denne aktuell for blankett? Hvis ikke kan den flyttes inn sammen med `opprettTaskForIverksettMotOppdrag`?

Reply:
Nei, er vel ikke det. Bra catch 👍